.prettierrc

```diff
@@ -8,7 +8,9 @@
         "tabWidth": 2,
         "useTabs": false,
         "singleQuote": true,
-        "bracketSpacing": true
+        "bracketSpacing": true,
+        "trailingComma": "es5",
+        "parser": "typescript"
```

Comment:
These were the default settings of prettier v2. I set them here like this to not pollute the PR with prettier-related changes.